app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable, :registerable,
```

Comment:
Only internal users only, they are already in system so not need to allow registration